fhir-tools/src/main/java/com/ibm/fhir/tools/CodeGenerator.java

```diff
@@ -913,12 +913,20 @@ private void generateClass(JsonObject structureDefinition, List<String> paths, C
                 }
             }
 
-            List<String> javadocLines = new ArrayList<>(Arrays.asList(getElementDefinition(structureDefinition, path).getString("definition").split(System.lineSeparator())));
+            List<String> javadocLines = new ArrayList<>(Arrays.asList(
+                    getElementDefinition(structureDefinition, path).getString("definition").split(System.lineSeparator())));
             if (isDateTime(structureDefinition)) {
                 javadocLines.addAll(Arrays.asList("", "If seconds are specified, fractions of seconds may be specified up to nanosecond precision (9 digits). However, any fractions of seconds specified to greater than microsecond precision (6 digits) will be truncated to microsecond precision when stored."));
             } else if (isInstant(structureDefinition) || isTime(structureDefinition)) {
                 javadocLines.addAll(Arrays.asList("", "Fractions of seconds may be specified up to nanosecond precision (9 digits). However, any fractions of seconds specified to greater than microsecond precision (6 digits) will be truncated to microsecond precision when stored."));
             }
+            if (!nested && hasMaturityLevel(structureDefinition)) {
+                String maturityLevel = getMaturityLevel(structureDefinition);
+                if (maturityLevel != null) {
+                    String maturityClass = getMaturityClass(maturityLevel);
+                    javadocLines.addAll(Arrays.asList("", "Maturity level: FMM" + maturityLevel + " (" + maturityClass + ")"));
```

Comment:
What to do if getMaturityClass(...) returns null?

Reply:
then it will print null in the javadoc, then go investigate why it was null (it should never be)